mkdocs.yml

```diff
@@ -136,15 +136,11 @@ nav:
         - Troubleshooting: synthetics/user-journey-scripts/troubleshooting.md
       - SSL Certificate Monitoring: synthetics/ssl-certificate-monitoring.md
       - CI/CD Integration:
-        - Overview: synthetics/ci-cd/overview.md
-        - Jenkins: synthetics/ci-cd/jenkins.md
-        - Travis CI: synthetics/ci-cd/travis-ci.md
-        - Circle CI: synthetics/ci-cd/circle-ci.md
-        - GitHub Actions: synthetics/ci-cd/github-actions.md
-        - Bitbucket Pipelines: synthetics/ci-cd/bitbucket-pipelines.md
-        - GitLab CI/CD: synthetics/ci-cd/gitlab-ci-cd.md
-        - Vercel: synthetics/ci-cd/vercel.md
-        - Netlify: synthetics/ci-cd/netlify.md
+        - Overview: synthetics/ci-cd/ci-cd-overview.md
+        - Installation: synthetics/ci-cd/ci-cd-installation.md
+        - Simple Workflow Example: synthetics/ci-cd/ci-cd-example-simple.md
+        - Complex Workflow Example: synthetics/ci-cd/ci-cd-example-complex.md
+        - Optional Check Run Fix: synthetics/ci-cd/ci-cd-check-run-fix.md
```

Comment:
@HSSalman The pages we've removed are also listed under Integrations, so we need to remove them from there as well.

Reply:
Nice catch, I missed that completely. Fixed.